packages/angular_devkit/build_angular/src/builders/browser-esbuild/schema.json

```diff
@@ -13,9 +13,16 @@
       }
```

Comment:
Moved the new `entryPoints` option into `InfrastructureSettings` of `browser-esbuild`, so it's out of the schema. I opted to leave the optional index in the schema as @clydin felt that was at least somewhat appropriate.

Unfortunately this got tricky to test because Architect doesn't have knowledge of this extra option, and I need to use the builder harness to get a valid context. I ended up adding just enough support for the builder harness to pass through this extra `InfrastructureSettings` (open to better naming suggestions) option into the builder. The actual production builder APIs still don't support passing through this value as that got more complicated and affected more of the public API which I would rather stay decoupled from here. Since we're calling `browser-esbuild` outside of Architect anyways, that functionality isn't actually needed yet.

Reply:
As mentioned in another comment, I dropped all the changes to `InfrastructureSettings` and handled them via a separate `InternalOptions` type exposed from a new `buildEsbuildBrowserInternal()` entry point.